controllers/managedocs_controller.go

```diff
@@ -136,3 +145,34 @@ func (r *ManagedOCSReconciler) setDesiredStorageCluster(
 
 	return nil
 }
+
+func (r *ManagedOCSReconciler) updateComponents(req ctrl.Request) ([]v1.ManagedOCSComponent, error) {
+	var components []v1.ManagedOCSComponent
+
+	// Checking the StorageCluster component.
+	component := v1.ManagedOCSComponent{
+		Name: "StorageCluster",
+	}
+
+	var scList ocsv1.StorageClusterList
+
+	if err := r.List(r.ctx, &scList); err != nil {
+		r.Log.Error(err, "error listing StorageCluster")
+		return nil, err
+	}
+
+	if len(scList.Items) < 1 {
+		component.Status = v1.ComponentNone
```

Comment:
Why do we need a status as all in this case, can't we just set the proper component status to nil?

Reply:
For debugging purposes I would prefer to see that the reconcile looked for the StorageCluster and found none at the time.

Reply:
I see 2 options:
1. Remove it from the status and add it to the log, the reason is that debug purposes should not inflect public API
2. Make it meaningful by changing the value from none to "Uninstalled" which has a valid meaning